html/inc/email.inc

```diff
@@ -21,14 +21,20 @@
 
 require_once("../inc/util.inc");
 require_once("../project/project.inc");
+require_once("../inc/token.inc");
 
 // send an email, using PHPMailer or not.
 //
-function send_email($user, $subject, $body, $body_html=null) {
+function send_email($user, $subject, $body, $body_html=null, $email_addr=null) {
     if (function_exists("make_php_mailer")) {
-        require_once("../inc/phpmailer/class.phpmailer.php");
+        require_once("../inc/PHPMailer/src/PHPMailer.php");
+        require_once("../inc/PHPMailer/src/SMTP.php");
```

Comment:
Please update https://boinc.berkeley.edu/trac/wiki/ServerIntro#PHPMailer so that has the correct changes so that projects know what to do.

Reply:
I have updated the instructions to point to 6.0.x PHPMailer and commited a code change to allow for both options, this way it won't break older version of PHPMailer.  https://github.com/BOINC/boinc/pull/2500/commits/4aa5acbc45f9e95197e3dd486bb1d8f2d8399790